cli/packages/prisma-cli-core/package.json

```diff
@@ -84,37 +84,26 @@
     "chokidar": "^1.7.0",
     "copy-paste": "^1.3.0",
     "cross-spawn": "^5.1.0",
-    "download-github-repo": "^0.1.3",
     "express": "^4.16.3",
     "express-request-proxy": "^2.0.0",
     "figures": "^2.0.0",
     "find-up": "^2.1.0",
     "fs-extra": "^5.0.0",
     "globby": "^6.1.0",
-    "graphcool-inquirer": "^1.0.3",
-    "graphql": "^0.13.1",
```

Comment:
This one might not be wanted, depending on whether or not you'll keep `graphql` as a peer dependency.